### PR TITLE
Added Bronze - Gold US Open Problems

### PR DESCRIPTION
**Pull request by jessechoe10**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [ ] I have tested my code.
- [ ] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [ ] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which includes the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

File: content/extraProblems.json [hunk 1]
```diff
@@ -835,6 +835,71 @@
...
+      }
+    },
+    {
+      "uniqueId": "usaco-1235",
+      "name": "Balancing a Tree",
+      "url": "http://usaco.org/index.php?page=viewproblem2&cpid=1235",
+      "source": "Gold",
+      "difficulty": "Hard",
+      "isStarred": false,
+      "tags": ["Tree DP", "Binary Search"],
```
Comment (bqi343) [inline content/extraProblems.json:897]: ^ neither of these are necessary?

File: content/extraProblems.json [hunk 2]
```diff
@@ -835,6 +835,71 @@
...
+      }
+    },
+    {
+      "uniqueId": "usaco-1229",
+      "name": "Alchemy",
+      "url": "http://usaco.org/index.php?page=viewproblem2&cpid=1229",
+      "source": "Bronze",
+      "difficulty": "Hard",
+      "isStarred": false,
+      "tags": ["Simulation", "Complete Search", "Recursion"],
```
Comment (bqi343) [inline content/extraProblems.json:858]: ^ no recursion necessary? also, how is it complete search

File: content/extraProblems.json [hunk 3]
```diff
@@ -835,6 +835,71 @@
...
+      }
+    },
+    {
+      "uniqueId": "usaco-1230",
+      "name": "Visits",
+      "url": "http://usaco.org/index.php?page=viewproblem2&cpid=1230",
+      "source": "Silver",
+      "difficulty": "Normal",
+      "isStarred": false,
+      "tags": ["SCC"],
```
Comment (bqi343) [inline content/extraProblems.json:871]: ^ not intended ...

File: content/extraProblems.json (hunk 1 above)
Comment (jessechoe10) [reply content/extraProblems.json:897]: what should i change this to?

File: content/extraProblems.json (hunk 3 above)
Comment (jessechoe10) [reply content/extraProblems.json:871]: should i put this in the functional graphs module?

**Comment by SansPapyrus683** [comment]
i don't really think just because something isn't necessary to solve a problem doesn't mean it shouldn't be included in the tags
as long as it's at least somewhat clear how the tag connects to the problem, it can be a viable tag

but anyways idrc jesse you do you

File: content/extraProblems.json (hunk 3 above)
Comment (bqi343) [reply content/extraProblems.json:871]: k

File: content/extraProblems.json (hunk 1 above)
Comment (bqi343) [reply content/extraProblems.json:897]: idk, just remove both of them?

File: content/2_Bronze/Intro_Complete.problems.json [hunk 4]
```diff
@@ -112,6 +112,19 @@
...
       }
     },
+    {
+      "uniqueId": "usaco-1231",
+      "name": "Subset Equality",
+      "url": "http://usaco.org/index.php?page=viewproblem2&cpid=1231",
+      "source": "Silver",
+      "difficulty": "Hard",
+      "isStarred": false,
+      "tags": ["Complete Search"],
```
Comment (bqi343) [inline content/2_Bronze/Intro_Complete.problems.json:122]: ^ huh?

File: content/3_Silver/Greedy_Sorting.problems.json [hunk 5]
```diff
@@ -226,6 +226,19 @@
         "kind": "USACO",
         "usacoId": "1158"
       }
+    },
+    {
+      "uniqueId": "usaco-1233",
+      "name": "Apple Picking",
```
Comment (bqi343) [inline content/3_Silver/Greedy_Sorting.problems.json:232]: wrong name

**Review by SansPapyrus683 — APPROVED** [review]

File: content/extraProblems.json [hunk 6]
```diff
@@ -835,6 +835,71 @@
...
+      }
+    },
+    {
+      "uniqueId": "usaco-1231",
+      "name": "Subset Equality",
+      "url": "http://usaco.org/index.php?page=viewproblem2&cpid=1231",
+      "source": "Silver",
+      "difficulty": "Normal",
+      "isStarred": false,
+      "tags": ["Prefix Sums"],
```
Comment (bqi343) [inline content/extraProblems.json:871]: no?

**Review by bqi343 — APPROVED** [review]

**Pull request merged**